nb2/api.py

```diff
@@ -41,3 +41,35 @@ def create_person():
     response.status_code = 201
 
     return response
+
+@bp.route('/quotes', methods=['POST'])
+def create_quote():
+    data = request.get_json() or {}
+    slack_user_id = data.get('slack_user_id')
+
+    target_person = Person.query.filter(Person.slack_user_id==slack_user_id).one()
```

Comment:
I think `one` raises an exception based on the docs. We might want `one_or_none` or `first` to return None and then handle the exception later. Unless maybe this did just end up doing what we want

Reply:
Yeah you're right, I just got an exception. I'll change it to one_or_none

Reply:
Done